apps/prairielearn/elements/pl-dataframe/pl-dataframe.py

```diff
@@ -142,7 +142,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         descriptors = frame.agg([get_dtype_function]).set_axis(
             ["dtype"], axis="index", copy=False
         )
-        other = descriptors.style.applymap(lambda v: "font-weight: bold;")
+        other = descriptors.style.map(lambda v: "font-weight: bold;")  # type: ignore
```

Comment:
I may have asked this before, but: does pyright support something like `// @ts-expect-error` in TypeScript? That is, a comment that says "this should be an error, but if this ever becomes *not* an error, make this comment itself an error"?

Reply:
There's an option to enable this globally, see https://github.com/python/mypy/issues/8655#issuecomment-1128937888

Reply:
Seems reasonable, I say go for it.

Reply:
Do you want to open a PR for that? Let me know if not, I can do it too.

Reply:
It's probably best suited to a separate PR. I'm at the gym right now so feel free to, I won't be able to open one till tomorrow probably.

Reply:
💪 I'll do it!